database/seed.js

```diff
@@ -10,21 +10,21 @@ const seedBundles = () => {
   for (let i = 0; i <= 99; i++) {
     bundleId = i + 1;
     cost = Math.floor(Math.random() * 4) * 5;
-    tierId1 = i*3 + 1;
-    tierId2 = i*3 + 2;
-    tierId3 = i*3 + 3;
+    tierId1 = i * 3 + 1;
+    tierId2 = i * 3 + 2;
+    tierId3 = i * 3 + 3;
     bundleData.push({
-      bundleId : bundleId,
-      tierId : tierId1,
+      bundleId: bundleId,
+      tierId: tierId1,
       tierOrder: 1
```

Comment:
I believe we are supposed to use trailing commas per the airbnb style guide. See also, e.g., lines 24, 28, 48, 54, 61, and 69.

Reply:
Thank you.